README.md

```diff
@@ -20,19 +20,19 @@ Sentiment Analysis using BERT model and Tensorflowjs
 * Place ./model folder under public directory
 
 ### How to run this app
-1. npm run dev
+1.
+```
```

Comment:
Don't put this section as a code block. Instead, remove the backticks and only backtick the code portions only like:

Run `npm install` first to install the required modules, then run `npm run dev`.

Reply:
Thanks. Looks better!